react/Button/Button.js

```diff
@@ -46,7 +51,11 @@ export default class Button extends Component {
   }
 
   render() {
-    const { color, className, loading, fullWidth, children, component, ...restProps } = this.props;
+    const { color, className, loading, fullWidth, children, component, group, ...restProps } = this.props;
+
+    if (group) {
+      return <div className={styles.group}>{children}</div>;
```

Comment:
This line is making me wonder whether it might be better to have a seperate `<ButtonGroup>` component instead?